packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/BackButton/BackButton.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
```

Comment:
This component is a prime candidate for splitting up into a container and component.